test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -308,6 +289,25 @@ private static Version readVersionsFromInfo(RestClient restClient, int numHosts)
     }
 
     public void test() throws IOException {
+        //skip test if it matches one of the blacklist globs
+        for (BlacklistedPathPatternMatcher blacklistedPathMatcher : blacklistPathMatchers) {
+            String testPath = testCandidate.getSuitePath() + "/" + testCandidate.getTestSection().getName();
+            assumeFalse("[" + testCandidate.getTestPath() + "] skipped, reason: blacklisted", blacklistedPathMatcher
+                .isSuffixMatch(testPath));
+        }
+
+        restTestExecutionContext.clear();
```

Comment:
I don't recall why the clear() call is here (between these two blocks), maybe it can stay in the before method close to `adminExecutionContext.clear();` ?